### ISSUE TITLE
right-click

### ISSUE THREAD

**Issue by the-wazz** [mathiasbynens/rel-noopener#4]
more of a comment than an issue. sry if this is the wrong place.
i was just playing with this and noticed that if i right-click > Open Link in New Tab (windows/firefox), the vulnerability does not happen. i wonder if this is the same across browsers and if the click event can be converted to a right-click or context menu action.

**Comment by mathiasbynens** [comment]
I can confirm that right-click → _Open Link in New Tab_ prevents the attack in Opera. However, ⌘ + clicking a link (which is supposedly equivalent to opening in a new tab) still allows it to happen.

Interesting observation!

**Comment by rbf** [comment]
Hey, I ended up here because `eslint` spotted a new error [`react/jsx-no-target-blank`](https://github.com/yannickcr/eslint-plugin-react/blob/4e80833ffbfbe8bb8ae30301cd9dbaf6591666e4/docs/rules/jsx-no-target-blank.md):

```
Using target="_blank" without rel="noopener noreferrer" is a security risk: see https://mathiasbynens.github.io/rel-noopener  react/jsx-no-target-blank
```

Similarly to the first comment, I realised that forcing a link (with or without `target="_blank"`) to open in a new tab (or window) by ⌘-clicking (or selecting "Open in new tab..." in the contextual menu) on it prevents the vulnerability from being exploitable. I've been able to validate both ways (⌘-clicking and with contextual menu option) on macOS High Sierra (v10.13.4) using Chrome (v67.0.3396.99), Safari (v11.1) and Firefox (v58.0.2).

This issue has been widely known for at least two years by now (see also this [HackerNews thread](https://news.ycombinator.com/item?id=11631292)) and I wonder if browsers have learned how to properly deal with this vulnerability, but I cannot find any confirmation.

Anyway, I thought it was worth leaving a comment, and this seemed an appropriate place for it. Sorry if not!

Cheers!
